io.github.dagargo.Elektroid.yaml

```diff
@@ -0,0 +1,42 @@
+id: io.github.dagargo.Elektroid
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: elektroid.sh
+finish-args:
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
```

Comment:
`--share=ipc` is necessary for x11 (and fallback)

Reply:
Added in 0826364769e099e7b5cd610b12fa362060abdebc.